### PR TITLE
Allow to specify chart type and y-domain with dashboard spec

### PR DESCRIPTION
**Pull request by jotak**
Available chart types are 'line', 'area' and 'bar'. Defaults to 'line'.
Y-domain refers to min & max on Y values.

Addresses both #41 and #14

### CODE REVIEW

**Comment by jotak** [comment]
How it could look like in Kiali (I've modified an existing dashboard just for showing possibilities) :
in kiali

Also, storybook updated:
storybook

File: web/pf4/src/components/KChart.tsx [hunk 1]
```diff
@@ -118,6 +118,14 @@ class KChart extends React.Component<KChartProps, State> {
       />
     );
     const scaleInfo = this.scaledAxisInfo(data);
+    const seriesBuilder = (this.props.chart.chartType === 'area')
```
Comment (israel-hdez) [inline web/pf4/src/components/KChart.tsx]: I sometimes tend to use the ternary like this. But, could we sacrifice shortness for clarity and use a `switch`? or, at least, include some indenting to make it clear the nesting?
Also, variable naming could be improved :)
Suggestion: line -> serie (and same for the keys)

Comment (jotak) [reply web/pf4/src/components/KChart.tsx]: I've changed the indentation style which I think makes it much more readable.
I'm not a big fan of switch to just assign a variable, especially in ts/js

**Comment by jotak** [comment]
thanks @israel-hdez

**Pull request merged**